src/app/test/local_plugins/manager_test.py

```diff
@@ -93,23 +127,26 @@ def test_exception(self, monkeypatch, tmp_path, loader):
         loader.load_plugins(path=tmp_path)
 
 
-@pytest.mark.skip
 class TestLoadNew(object):
-    def test_empty_path(self, tmp_path, manager):
-        manager.load_new(path=tmp_path)
-        assert manager._runners == {}
+    def test_path_with_no_plugins(self, tmp_path, manager):
```

Comment:
Since these are adjacent in functionality to what needed to be tested for this ticket, it was low-effort to fix them and reuse the new logic.